components/blocks/FilesToCards.vue

```diff
@@ -58,12 +60,14 @@
             <a
               v-else-if="link.target.startsWith('mailto')"
               :href="link.target"
-              class="m-1
-                  link-item
-                  d-button
-                  has-background-link has-text-white has-text-weight-semibold
-                  is-size-5
-                  link-button"
+              class="
+                m-1
+                link-item
+                d-button
+                has-background-link has-text-white has-text-weight-semibold
```

Comment:
I'd collapse the first if and else into one element `v-if="link.target.startsWith('mailto') || link.target.startsWith('http')"` so the styles don't get repeated.